l10n_co_geo/data/res_bank.xml

```diff
@@ -0,0 +1,864 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+  <data noupdate="1">
+    <!-- <record model="res.bank" id="bank0">
```

Comment:
I think, l'll opt for the second, who has a bank account with Allianz Seguros de Vida SA? 😄 I think this can be marked as solved.